src/plugins/world_control/WorldControl.cc

```diff
@@ -306,6 +306,18 @@ void WorldControl::OnPause()
   this->dataPtr->SendEventMsg(msg);
 }
 
+/////////////////////////////////////////////////
+void WorldControl::OnReset()
+{
+  gz::msgs::WorldControl msg;
+  auto msgReset = new gz::msgs::WorldReset();
```

Comment:
nit: already in the `gz` namespace
```suggestion
  msgs::WorldControl msg;
  auto msgReset = new msgs::WorldReset();
```

Reply:
https://github.com/gazebosim/gz-gui/pull/476/commits/dbd8c74f31839dd4186bc074e2d0c30ed4748f1c